cobspec/src/main/java/com/saralein/cobspec/Main.java

```diff
@@ -43,6 +43,8 @@ private static Routes setupRoutes(Path root) {
                     .addRoute("/method_options", Methods.HEAD.name(), new DefaultController())
                     .addRoute("/method_options2", Methods.OPTIONS.name(), new OptionsController(Methods.allowGetAndOptions()))
                     .addRoute("/method_options2", Methods.GET.name(), new DefaultController())
+                    .addRoute("/tea", Methods.GET.name(), new DefaultController())
+                    .addRoute("/coffee", Methods.GET.name(), new CoffeeController())
```

Comment:
A potential future refactoring could be to update `addRoute` to accept a `Method` instead of a `String`. I think that would enable you to do

```java
new RoutesBuilder(/* ... */)
    .addRoute("/foo", Methods.GET, new MyController());
```

Reply:
Definitely want to fix this.  I started to do it on this branch, but then decided it might make more sense to do it on my basic auth branch.  So, it should pop up in the future.